bench/hamming-cleanup/quadratic.fpcore

```diff
@@ -0,0 +1,127 @@
+; -*- mode: scheme -*-
+
+(FPCore (a b c)
+ :name "quadp (p42, positive)"
+ ;:herbie-expected #f ; Fails in single precision due to bad samples
+ :herbie-target
+ (let ([sqtD
+  (let ([x (* (sqrt (fabs a)) (sqrt (fabs c)))])
+    (if (or (and (> a 0) (> c 0)) (and (<= a 0) (<= c 0)))
+      (let ([acx
+        (if (and (> (fabs a) 1) (> x 1))
+                (* (/ (fabs a) x) (fabs c))
+                (* (fabs a) (/ (fabs c) x)))])
+      (let ([acxe 
+        (if (and (> (fabs a) 1) (> x 1))
+              (+ (fma (/ (fabs a) x) (- (fabs c)) acx) (* (/ (fma (/ (fabs a) x) x (- (fabs a))) x) (fabs c)))
+              (+ (fma (- (fabs a)) (/ (fabs c) x) acx) (* (/ (fma (/ (fabs c) x) x (- (fabs c))) x) a)))])
```

Comment:
I think this kind of double-double trick is probably more than we should expect from Herbie, or if did do this it'd probably be through a double-double precision, not through manually unrolling like this. (Though this was necessary for the Racket impl.)

Reply:
Without double-double stuff we wouldn't need `acx` or `acxe` or `dp`, we'd just return `(* (sqrt (- (fabs b) x)) (sqrt (+ (fabs b) x))` if `(> (fabs b) x)`.

Reply:
What would we return if (> (fabs b) x) was not true?

Reply:
Oh wait is that just NAN?